pkg/webui/components/dropdown/index.js

```diff
@@ -56,6 +57,11 @@ const DropdownItem = ({ icon, title, path, action, exact, showActive, tabIndex }
       {iconElement}
       <Message content={title} />
     </button>
+  ) : external ? (
+    <Link.Anchor href={path} external tabIndex={tabIndex}>
```

Comment:
Please a new smoke test to verify that the user is able to reach the profile settings page and submit the form

Reply:
@bafonins, I tried adding this bit to cypress test, to submit the form (copied from account app tests), but unfortunately it fails, as it can't find `General Settings`, though I see in screenshots, that form is there. Could you help me figure out what could be the problem?

```
cy.findByText('General settings', { selector: 'h3' })
    .closest('[data-test-id="collapsible-section"]')
    .within(() => {
      cy.findByRole('button', { name: 'Expand' }).click()
    })
  
  cy.findByRole('button', { name: 'Save changes' }).click()
  cy.findByTestId('error-notification').should('not.exist')
  cy.findByTestId('toast-notification')
    .should('be.visible')
    .findByText('Profile updated')
    .should('be.visible')
```

Reply:
Hm, thats weird. Please try this:

```js
    it('succeeds navigating to Account App', () => {
      const user = {
        ids: { user_id: 'test-account-app-user' },
        primary_email_address: 'test-account-app-user@example.com',
        password: 'ABCDefg123!',
        password_confirm: 'ABCDefg123!',
        name: 'Test Account App User',
      }

      cy.createUser(user)
      cy.loginConsole({ user_id: user.ids.user_id, password: user.password })
      cy.visit(Cypress.config('consoleRootPath'))

      cy.get('header').within(() => {
        cy.findByTestId('profile-dropdown')
          .should('contain', user.name)
          .as('profileDropdown')

        cy.get('@profileDropdown').click()

        cy.get('@profileDropdown')
          .findByText(/Profile settings/)
          .should('have.attr', 'href', '/oauth/profile-settings')
          .should('have.attr', 'target', 'blank')

        cy.get('@profileDropdown')
          .findByText('Profile settings')
          .then(link => {
            cy.visit(link.prop('href'))
            cy.location('pathname').should('eq', '/oauth/profile-settings')
          })
      })

      cy.findByText('General settings', { selector: 'h3' })
        .closest('[data-test-id="collapsible-section"]')
        .within(() => {
          cy.findByRole('button', { name: 'Expand' }).click()
        })

      cy.findByRole('button', { name: 'Save changes' }).click()
      cy.findByTestId('error-notification').should('not.exist')
      cy.findByTestId('toast-notification')
        .should('be.visible')
        .findByText('Profile updated')
        .should('be.visible')
    })
```
Seems to be working just fine.
Could you also move this this to `smoke/profile-settings` folder as a separate e2e test?